Client/Frontend/Toolbar+URLBar/ConnectionStatusImage+WebSiteConnectionTypeStatus.swift

```diff
@@ -0,0 +1,18 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Shared
+
+extension ConnectionStatusImage {
```

Comment:
I think personally I would have kept `getForStatus` as part of the other file and not a separate extension since it is an integral part of it and only ever used with that (the other variables could even be private).

Even so this is not major and we do not have any defined styling guidelines, so I'm also ok with leaving it like this.

Reply:
Thanks for that!
My thought was that `ConnectionStatusImage` and `WebsiteConnectionTypeStatus`, despite being utilized to solve one scenario, they are conceptually two different things.
`ConnectionStatusImage` strictly belongs to the UI domain while `WebsiteConnectionTypeStatus` might be utilized for non-UI specific business logic.